addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -4,6 +4,28 @@ import options from 'web_editor.snippets.options';
 
 options.registry.WebsiteEvent.include({
 
+    /**
+     * @override
+     */
+    async start() {
+        const res = await this._super(...arguments);
+        this.currentWebsiteUrl = this.ownerDocument.location.pathname;
+        this.eventId = this._getEventObjectId();
```

Comment:
These is already set in the parent so I'd say either use the attribute from `this`, or set a `const`. As we don't actually care about setting this for the class in this override.